airflow/bin/cli.py

```diff
@@ -614,6 +614,16 @@ def next_execution(args):
         print(None)
 
 
+@cli_utils.action_logging
+def rotate_credentials(args):
```

Comment:
Code looks good, only comment is that I think this name needs changing - even knowning what the PR was for I thought this was about rotating the credentials _in_ the connection.

How about `reencrypt_connections`, `rotate_fernet_key`, or `rotate_encryption_keys`?